README.md

```diff
@@ -30,10 +30,81 @@ How to run it
 -------------
 To preview patch generated test cases:
 ```bash
-$ kpet --db . run print-test-cases 001.patch
+$ kpet --db <path-to-kpet-db> run print-test-cases 001.patch
 ```
 
 To generate complete beaker xml:
 ```bash
-$ kpet --db . run generate --description 'skt ##KVER##' -a aarch64 -k '##KPG_URL##' -t upstream 001.patch
+$ kpet --db <path-to-kpet-db> run generate --description 'skt ##KVER##' -a aarch64 -k '##KPG_URL##' -t upstream 001.patch
```

Comment:
You can omit `--db` if your current directory is the database's root. Could you rewrite this section to avoid introducing the option? This way it will be easier to digest, I think.